src/main/java/com/student/user/app/controllers/item/AddStudentController.java

```diff
@@ -207,12 +241,27 @@ private boolean isValidInput(String fieldName, String input) {
             case "Contact Number":
                 return Pattern.matches("^[0-9]{11}$", input);
 
+            case "Birthday":
+                return isValidBirthday(input);
+
 
             default:
                 return false;
         }
     }
 
+    private boolean isValidBirthday(String birthdayString) {
+        LocalDate birthdayDate = LocalDate.parse(birthdayString);
+        if (birthdayDate == null) {
+            return false;
+        }
+
+        LocalDate currentDate = LocalDate.now();
+        LocalDate minValidDate = currentDate.minusYears(17);
```

Comment:
What about the possibility that a student may enter college earlier due to acceleration?